app/models/live_metric.rb

```diff
@@ -8,18 +8,20 @@ def self.find(*args)
     validate_raw_query(raw_query)
     processed = process_conditions(raw_query[:conditions])
     resource = fetch_resource(processed[:resource_type], processed[:resource_id])
+    ext_options = raw_query[:ext_options]
+    filtered_cols = ext_options ? ext_options[:only_cols] : nil
     if resource.nil?
       []
     else
-      filter_and_fetch_metrics(resource, raw_query[:ext_options][:only_cols], processed[:start_time],
+      filter_and_fetch_metrics(resource, filtered_cols, processed[:start_time],
```

Comment:
filtered_cols can still be nil based on line 12, is that ok?
btw pls remind me, what are the filtered_cols and ext_options for?